.github/workflows/bagua-python-package-check.yml

```diff
@@ -29,3 +29,98 @@ jobs:
       run: |
         rm -rf bagua bagua_core
         pytest -s --timeout=300 --timeout_method=thread
+  build-ten:
```

Comment:
I wonder why adding so many steps on different containers here, rather than changing above `**pytorch-1.9.0**` directly?

Reply:
@wangraying This is just to test if it can run properly on different versions of PyTorch. If this PR needs to be merged, the final testing environment will either PyTorch 1.9.0 or PyTorch 1.13.0.
If the test case will to be changed to PyTorch 1.13.0 version later, we can still keep the PyTorch 1.9.0 testing environment here.

Reply:
I just checked pytorch lightning's ci, their [pytest cpu test](https://github.com/Lightning-AI/lightning/blob/master/.github/workflows/ci-tests-pytorch.yml) supporting multiple torch versions, but gpu test only running on pytorch 1.13. Maybe we could follow their convention too, or just running all ci test on pytorch 1.13.